c10/core/TensorImpl.cpp

```diff
@@ -227,119 +227,170 @@ void TensorImpl::HandleResize() {
   }
 }
 
-bool TensorImpl::compute_contiguous() const {
+template <typename T>
+bool_is_contiguous _compute_contiguous(
```

Comment:
i can't read this code. what's going on here? is this syntax?

Reply:
i guess bool_is_contiguous is a type, but i can't find where it's defined..

Reply:
oh nvm github was hiding the .h from me so i couldn't search it

Reply:
top of TensorImpl.h. It's a wrapper around bool